src/modeles/rolesResponsabilites.js

```diff
@@ -68,7 +68,7 @@ class RolesResponsabilites extends InformationsHomologation {
     return this.partiesPrenantes.developpementFourniture()?.nom || '';
   }
 
-  descriptionActeursHomologation() {
+  descriptionResponsables() {
     const acteurHomologationDecrit = (role, description) => ({ role, description });
     const description = (acteur) => acteur.nom + (acteur.fonction ? ` (${acteur.fonction})` : '');
     const acteursSpecifiques = () => this.acteursHomologation.tous()
```

Comment:
Il me semble que par cohérence il faudrait aussi changer ce `this.acteursHomologation` en `this.responsables`… ce qui tire une pelote de changements un peu plus conséquents que ceux présents ici (notamment des migrations en base de données).

Reply:
C'est une partie très compliquée et tu poses beaucoup de bonnes questions. Je ne suis personnellement pas pleinement satisfait du résultat mais on s'est arrêté là-dessus. Je ne suis pas contre en re-discuter. Le gros sujet que nous avons eu était de savoir si nous conservions cette idée de 2 onglets ou si nous déplacions les rôles "acteurs de l'H / responsables" dans la partie "avis sur le dossier" car c'est surtout à ce moment là que savoir qui est en charge de quoi et ce qu'ils/elles pensent est important. Nous avons pour le moment tranché sur le fait de laisser les 2 ensemble.
- Concernant le nom de rubrique, j'aime beaucoup la notion de "cartographie" qui renvoie à des mesures que nous recommandons dans le référentiel et si on parle seulement d'écosystème j'ai peur que nous perdions les novices en SSI. So far, "Acteurs" a été retenu. Personnellement, je trouve que "parties prenantes" marcherait mieux (tout seul d'ailleurs potentiellement, sans "cartographie"), on se l'est juste interdit car ce n'est pas ce qui est dit dans le guide en 9 étapes (et peut-être la réglementation). Si je m'en réfère aux définitions que je trouve sur internet, ça donne "Une partie prenante désigne tout acteur (individu, organisation, groupe) concerné par un projet, une décision ou action, c’est-à-dire dont les intérêts sont affectés d’une façon ou d’une autre par sa mise en place. Les parties prenantes peuvent être internes ou externes".
- Concernant "Acteurs de l'homologation" > l'élément clé ici est que nous souhaitons déconnecter cette liste de la démarche d'homologation pour parler du service (même si reste l'autorité d'homologation). Et "acteurs" tout seul n'était pas suffisant. L'intérêt de "Responsables" est qu'il s'agit de parler de gens ayant une part de responsabilité dans le projet. Ce qui les distingue des entités/services/personnes qui contribuent simplement à son fonctionnement. Après la limite du mot "responsable" est que cela pourrait faire penser aux "chefs/managers". Reparlons-en.
- Pour finir nous avons retenu "Contributeurs" pour parler de toutes celles et tous ceux qui participent à faire tourner le service. Je ne suis pas pleinement satisfait non plus.
> En conclusion, OK pour mettre la PR en attente et refaire un bout d'atelier là-dessus et embarquer MSN au passage qui a demandé un droit de regard (j'avais zappé).

Reply:
A chaud, je me dis qu'on pourrait peut-être avoir
- En titre de rubrique "Parties prenantes"
- L'onglet "Gouvernance"
- L'onglet "Fonctionnement"